libs/langchain/tests/integration_tests/document_loaders/test_pdf.py

```diff
@@ -150,14 +150,14 @@ def test_mathpix_loader() -> None:
                 "https://amazon-textract-public-content.s3.us-east-2.amazonaws.com"
                 "/langchain/alejandro_rosalez_sample_1.jpg"
             ),
-            ["FORMS", "TABLES"],
+            ["FORMS", "TABLES", "LAYOUT"],
             1,
             False,
         ),
         (str(Path(__file__).parent.parent / "examples/hello.pdf"), ["FORMS"], 1, False),
         (
             "s3://amazon-textract-public-content/langchain/layout-parser-paper.pdf",
-            None,
+            ["FORMS", "TABLES", "LAYOUT"],
```

Comment:
Should we keep a test without these features?

Reply:
added one without FORMS and LAYOUT and one without any features

Reply:
@anjanvb 
This still doesn't have a test without any features.

Reply:
@3coins : (str(Path(__file__).parent.parent / "examples/hello.pdf"), [], 1, False), has not features